frontend/src/component/changeRequest/ChangeRequest/Changes/Change/StrategyChange.tsx

```diff
@@ -118,6 +127,19 @@ export const StrategyChange: VFC<{
         environmentName
     );
 
+    const isStrategyAction =
+        change.action === 'addStrategy' || change.action === 'updateStrategy';
+
+    const featureStrategyVariantsDisplay =
```

Comment:
I'd consider adding a test for StrategyChange checking if variants are displayed. I have a strategy that I add tests for every UI component that I touch if there's any logic inside.